pkgs/development/libraries/libblockdev/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gtk-doc, libxslt, docbook_xsl
+, python3, gobjectIntrospection, glib, libudev, kmod, parted, cryptsetup
+, devicemapper, dmraid, utillinux, libbytesize, libndctl, nss, volume_key
+}:
+
+let
+  version = "2.17";
+in stdenv.mkDerivation rec {
+  name = "libblockdev-${version}";
+
+  src = fetchFromGitHub {
+    owner = "storaged-project";
+    repo = "libblockdev";
+    rev = "${version}-1";
+    sha256 = "14f52cj2qcnm8i2zb57qfpdk3kij2gb3xgqkbvidmf6sjicq84z2";
+  };
+
+  outputs = [ "out" "dev" "devdoc" ];
+  outputBin = "dev";
```

Comment:
What's the purpose of this? The dev output doesn't seem to contain any binaries.

Reply:
Must have been a leftover I blindly copied over, my bad. Removed from here and `volume-key`.